### PR TITLE
Mirage decal applied to arena on activation

### PR DESCRIPTION
**Pull request by ClaysGit**
Quick and simple Mirage decal applied to the Multiroller Arena on activation

I made the Mirage image to have a transparent background. Image and GIMP files attached. Currently the URL points to the image in my own Steam cloud.
[Mirage Token Transparent.zip](https://github.com/darrellanderson/CrLua/files/5692923/Mirage.Token.Transparent.zip)

Also threw in minor bug fix for Vuil'Raith Hero.

### CODE REVIEW

**Comment by ClaysGit** [comment]
Just using the planet position worked really well

image

File: Objects/AutoFillMultiRollerArena.ttslua [hunk 1]
```diff
@@ -40,6 +40,24 @@ function onPlayerTurnEnd(player_color_end, player_color_next)
...
+    local decals = {}
+
+    for _, planet in ipairs(system.planets or {}) do
+        if planet.planetDecal then
+            table.insert(decals, {
+                name = planet.name .. ' decal',
+                url = planet.planetDecal,
+                position = { x = planet.position.x, y = 0.1, z = planet.position.z },
+                rotation = { x = 90, y = 180, z = 0 },
+                scale = { x = 2.3, y = 2.3, z = 1 }, -- TODO: rework Mirage image so this can be 1,1,1
```
Comment (darrellanderson) [inline Objects/AutoFillMultiRollerArena.ttslua:53]: actually no, decal "scale" is not (this cost me a solid day the first time using decals).  decal "scale" is actually the world space size in which to fit the image, completely independent of the image itself.

File: Objects/TI4_Helpers/TI4_FactionHelper.ttslua [hunk 2]
```diff
@@ -1466,8 +1465,8 @@ function _carrionHeroAbilityCoroutine()
     end
     assert(factionTokenName and factionColor, 'Dimensional Anchor: Trying to invoke Hero Ability, but no faction with "It Feeds on Carrion" is at the table.')
 
-    --Find each dimensional tear token
-    local dimensionalTearName = "Vuil'raith Cabal Gravity Rift Token"
+    --Find each dimensional tear token (Cabal only)
+    local dimensionalTearName = "Tear Token (Cabal)"
```
Comment (darrellanderson) [inline Objects/TI4_Helpers/TI4_FactionHelper.ttslua:1469]: oops, nice catch

**Review by darrellanderson — APPROVED** [review]
cool!  I'll approve, merge, and tweak your decal scale comment.

**Pull request merged**